Content.Server/GameTicking/Rules/GameRulePrototype.cs

```diff
@@ -1,10 +1,14 @@
+using Content.Server.GameTicking.Rules.Configurations;
 using Robust.Shared.Prototypes;
 
 namespace Content.Server.GameTicking.Rules;
 
 [Prototype("gameRule")]
 public sealed class GameRulePrototype : IPrototype
 {
-    [IdDataFieldAttribute]
+    [IdDataField]
     public string ID { get; } = default!;
```

Comment:
?

Reply:
i have no goddamn idea this is all the work of #8539

Reply:
Aight will move over there.